File: frontend/src/services/Post.ts
export interface CompletePost
{
    idAuthor : number
    authorName : string
    content : string
    createdAt : Date
    crowds : number
    comments : number
    idPost : number
    photo : string
    idForum : number
    forumName : string
    fkPost : number
    title : string
}

export interface Post
{
    id : number
    author : number
    title : string
    content : string
    createdAt : Date
    crowds : number
    comments : number
    idPost : number
    idForum : number
}

File: backend/Model/Post.cs
﻿using System;
using System.Collections.Generic;

namespace backend.Model;

public partial class Post
{
    public int Id { get; set; }

    public int? Author { get; set; }

    public string? Title { get; set; }

    public string? Content { get; set; }

    public string? Anex { get; set; }

    public DateTime? CreatedAt { get; set; }

    public int? IdPost { get; set; }

    public int? IdForum { get; set; }

    public virtual Forum? IdForumNavigation { get; set; }

    public virtual Post? IdPostNavigation { get; set; }

    public virtual ICollection<Post> InverseIdPostNavigation { get; set; } = new List<Post>();

    public virtual ICollection<UserXlike> UserXlikes { get; set; } = new List<UserXlike>();
}
